### PR TITLE
Adding beforeAll and afterAll

### PR DESCRIPTION
**Pull request by kyle-long**

### CODE REVIEW

File: lib/promises.helper.js [hunk 1]
```diff
@@ -80,11 +80,19 @@ function patchAllMethods(obj) {
             patchMethod(1, obj, methodName);
         }
     });
+
+    /* beforeAll and afterAll were added in jasmine 2.1
```
Comment (AbsentSemicolon) [inline lib/promises.helper.js]: jasmine should be capitalized.

File: spec/mock/jasmine-prototypal-mock.js [hunk 2]
```diff
@@ -3,6 +3,9 @@
 module.exports = () => {
     var mock;
 
+    /* This is the way the jasmine object will be
```
Comment (AbsentSemicolon) [inline spec/mock/jasmine-prototypal-mock.js]: jasmine should be capitalized.

Comment (kyle-long) [reply spec/mock/jasmine-prototypal-mock.js]: f69bacc074b13849d634fbe6fbb0bd2e5ef28669

File: lib/promises.helper.js (hunk 1 above)
Comment (kyle-long) [reply lib/promises.helper.js]: f69bacc

**Comment by AbsentSemicolon** [comment]
Looks good.

**Pull request merged**